localization/si/README.md

```diff
@@ -0,0 +1,69 @@
+<!-- පහත පේළිය හිස් රේඛාවක් විය යුතුය C: (එය kramdown නොවන බැවිනි
+     ශීර්ෂයකට පෙර හිස් රේඛාවක් අවශ්‍ය බව බුද්ධිමත් හා ආදරණීයයි
+     එය එසේ පෙන්වන්න, උදා. වෙබ් අඩවිය) -->
```

Comment:
This metadata part needs to be in English. See for example here: https://raw.githubusercontent.com/iluwatar/java-design-patterns/master/localization/es/abstract-document/README.md

Reply:
changed to English.